csrankings.csv

```diff
@@ -9187,6 +9190,7 @@ Joon Hee Han,POSTECH,http://cvlab.postech.ac.kr/jhhan,_mMeOTgAAAAJ
 Joos C. A. M. Buijs,TU Eindhoven,https://www.tue.nl/en/university/departments/mathematics-and-computer-science/the-department/staff/detail/ep/e/d/ep-uid/20020361,NOSCHOLARPAGE
 Joost Batenburg,CWI,https://www.cwi.nl/people/1742,6WpX7xUAAAAJ
 Joost Broekens,Leiden University,http://www.joostbroekens.com,vw2QkREAAAAJ
+Joost van de Weijer,Autonomous University of Barcelona,http://www.cvc.uab.es/LAMP/joost/,Gsw2iUEAAAAJ

```

Comment:
Senior scientist, not faculty.

Reply:
Most of these members belong to the computer vision center (CVC) in UAB, maybe the system in CVC is different. Those members are PIs in different research groups, all can supervise PhD student and postPhD solo.

Reply:
It doesn't matter: CSrankings is currently limited to tenure-track faculty.